packages/ia-icons/README.md

```diff
@@ -0,0 +1,31 @@
+# ia-icons: SVG icon library for use in archive.org components
+
+## Usage
+
+```js
+import { IconClose } from './src/ia-icons';
+
+customElements.define('icon-close', IconClose);
+```
+
+```html
+<icon-close></icon-close>
```

Comment:
Is the intended use to define each icon individually like this?

I think it would help icon discoverability and consumption if it were packaged under a single component with an enumerated list of options. ie:
```
<ia-icon icon="close"></ia-icon>
```
I'm not sure if there's a good way to do an enum of the icon options with JS, but it'd be great to have some reference of what icons are available without having to open up the readme.

Reply:
I was thinking that there would end up being so many icons that even if you only wanted one icon in your project, you'd be bringing the rest along with it.

As far as reference is concerned, if you run the server the index will render a matrix of all of the icons defined with a random fill and stroke color.
![Screen Shot 2020-05-21 at 3 30 39 PM](https://user-images.githubusercontent.com/39553/82598071-0cb83100-9b78-11ea-9af9-465f5c4ac698.png)

Reply:
Regarding only bringing a single icon in, it looks like they all get imported in that top-level `Icon` class so you're already importing them all.

Hmm it'd be nice to not have to run the project to see the reference. We'll probably be using this library while working on other things so spinning this up for reference.

Reply:
I was thinking https://archive.org/help/coding.php would have the icons rendered once we've included it in Petabox.